### ISSUE TITLE
Window movement while focusing comment box

### ISSUE THREAD

**Issue by svenuthe** [jupyter/nbgrader#982]
### Operating system
Fedora

### `nbgrader --version`
python 3.5.3
nbgrader 0.5.4

### `jupyterhub --version` (if used with JupyterHub)
4.4.0

### `jupyter notebook --version` 
5.4.1

### Expected behavior
After focusing the comment textbox, the browser window shouldn't move.

### Actual behavior
The Browser window moves to the head of the answer.
Only recognizable, if answer is very long.

### Steps to reproduce the behavior
As evaluator, open submission in formgrader.
Click into comment box below a very long answer.

**Comment by jhamrick** [comment]
This is generally expected behavior for the window to move (the idea is so that you can tab between the answers and they scroll into view as you do so). Is there a bug with this behavior that prevents proper usage of nbgrader (i.e. does the comment box get scrolled off the screen?) If you could provide some more details that would be helpful!

**Comment by svenuthe** [comment]
>does the comment box get scrolled off the screen?
Yes, if you click in the comment textbox the window scrolls up to the head of the answer and the comment box scrolls out of the view.

**Comment by jhamrick** [comment]
Ah, I see. Yes, that is a problem.
